client/components/ResponsiveFlex/ResponsiveFlex.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropsWithChildren, useEffect } from 'react';
+
+interface Props {
+  breakpoint: number;
+}
+
+function ResponsiveFlex({ breakpoint, children }: PropsWithChildren<Props>) {
+  const containerRef = React.createRef<HTMLDivElement>();
+
+  useEffect(() => {
+    if (breakpoint) {
+      if (document.head.querySelector(`[data-flex-breakpoint="${breakpoint}"]`)) {
+        containerRef.current.className = `responsive-flex--${breakpoint}`;
```

Comment:
We shouldn't set the class like this (via traditional DOM methods). Instead at the begging of the function we can directly have `const responsiveClassName = 'responsive-....'`  (because we know what class this component will have depending on the breakpoint).

And when rendering `<div ref={containerRef} className={responsiveClassName }>`.

PS: let's also allow this component to receive classes as props. For example, we want to be able to do this

```
<ResponsiveFlex breakpoint={500} className="mt-2 mb-4">
```

and the rendered container should have both the breakpoint-related class and `mt-2 mb-4`.